ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -5,6 +5,7 @@
 from styx_msgs.msg import Lane, Waypoint
 
 import math
+#import sys
```

Comment:
I don't think this is needed

Reply:
to be clear, which line of code do you not think is not needed? Line 8 is commented out, but I can delete it.

Reply:
Correct, my mistake